src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue19609.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Drawing;
+using NUnit.Framework;
+using NUnit.Framework.Legacy;
+using OpenQA.Selenium.Interactions;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+public class Issue19609 : _IssuesUITest
+{
+	public Issue19609(TestDevice device) : base(device) { }
+
+	public override string Issue => "Navigating Back to FlyoutPage Renders Blank Page";
+
+	[Test]
+	[Category(UITestCategories.CollectionView)]
+	public void CanTapButtonOnEmptyView()
```

Comment:
This test is failing on Android, iOS and Catalyst.

`at Microsoft.Maui.TestCases.Tests.Issues.Issue19609.CanTapButtonOnEmptyView() in /_/src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue19609.cs:line 19`

Could you check that the button is on the screen? You can use the `timeout` parameter to wait for one second for example.